packages/cli/openapi-parser/src/schema/convertNumber.ts

```diff
@@ -80,6 +80,7 @@ export function convertNumber({
             nameOverride,
             generatedName,
             primitive: PrimitiveSchemaValueWithExample.int64({
+                default: _default,
```

Comment:
would be great to add a fixture (or update existing) to check that the parsing for longs and booleans work?